### DISCUSSION TITLE
Can github config be changed to show main by default.

### DISCUSSION THREAD

**Discussion by bradh** [ngageoint/six-library#745] [General]
Thanks.

**Comment by JDanielSmith** [comment]
Not yet :-(

**Comment by bradh** [comment]
Just to confirm - github does allow it. Settings -> Default branch.

![image](https://github.com/ngageoint/six-library/assets/174642/39ff0c05-519b-46b8-8dfb-a1039d823226)


Just need to have permissions and policy in place...

Just without that change, it looks like things are fairly dead despite the activity on `main`.
